### PR TITLE
Testing Components

### PR DESCRIPTION
**Pull request by Bigizi**
**Math-Magician-Project**

I added a test to each component
The test files added are:
Calculator.js
Home.js
Quote.js

### CODE REVIEW

**Review by Whoistolu — COMMENTED** [review]
Hi Team,
@Bigizi and @crispin12 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a pair programming project, that needs to have equal or almost equal commits from both parties but unfortunately, only one person has contributed to this project so far. please fix this. Also, the README file does not have the details of the other partner in the Authors section. Kindly fix this as well before requesting another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import { render, screen } from '@testing-library/react';
+import Calculator from './Calculator';
+
+describe('Calculator', () => {
+  it('renders Calculator component', () => {
+    render(<Calculator />);
+    const buttonArr = screen.getAllByRole('button');
+    expect(buttonArr.length).toBe(19);
+  });
+});
```
Comment (GzimAsani) [inline src/components/Calculator.test.js:11]: - Good job adding tests for components and snapshots :clap: 
- [ ] But according to the requirements you need to add tests all the functionalities & scenarios of the operate.js and calculate.js modules should be covered using Jest. 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, **TEAM**            :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with snapshots :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Wes-Isaac — APPROVED** [review]
Hi @Bigizi & @crispin12,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Professional README 👍.
- Excellent communication skills 💯.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**